tests/fixtures.py

```diff
@@ -223,13 +228,30 @@ def sample(request):
     return resolve_sample(request.param)
 
 
-def get_function(extractor, fva):
+def get_function(extractor, fva, force=True):
```

Comment:
i'm not a huge fan of how this function behaves implicitly (by creating the function if it doesn't exist, which doesn't match how capa works from the cli). 

i'd recommend moving the `make_function` calls to a separate procedure, maybe invoked by the `get_workspace` function used by tests. this way, a user can easily find all the fixups we do to support testing.

Reply:
for example, within `3b13b_extractor()`, check `isinstance(extractor, VivisectExtractor)` and if so, do the appropriate `make_function` calls before returning it.